frontend_tests/node_tests/compose_validate.js

```diff
@@ -673,22 +672,30 @@ test_ui("warn_if_private_stream_is_linked", ({mock_template}) => {
     }
 });
 
-test_ui("warn_if_mentioning_unsubscribed_user", ({override, override_rewire, mock_template}) => {
+test_ui("warn_if_mentioning_unsubscribed_user", ({override, mock_template}) => {
     override(settings_data, "user_can_subscribe_other_users", () => true);
 
-    let mentioned = {
+    let mentioned_user = {
```

Comment:
The name made sense to me because the line `people.add_active_user(mentioned_user);`, but I can change it to `mention_details` if you think that is clearer.

Reply:
Yeah, I think that's better -- generally a thing that says "user" should be some version of a user object.